collectors/cochrane_reviews/parser.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import uuid
+try:
+    from lxml import etree
+except ImportError:
+    import xml.etree.ElementTree as etree
+from .record import Record
+
+
+def parse_record(url, review_file):
+    tree = etree.parse(review_file)
+    study_robs = []
+    studies = []
+
+    # Get risk of bias
+
+    root = tree.getroot()
+    doi_id = root.attrib.get('DOI', '')
+    quality_item_data_entries = tree.findall('//QUALITY_ITEM_DATA_ENTRY')
+    for quality_item_data_entry in quality_item_data_entries:
+        study_rob = {
+            'study_id': quality_item_data_entry.attrib['STUDY_ID'],
+            'modified': quality_item_data_entry.attrib.get('MODIFIED', ''),
+            'result': quality_item_data_entry.attrib['RESULT'],
+            'group_id': quality_item_data_entry.attrib.get('GROUP_ID', ''),
+            'group_name': '',
+            'result_description': quality_item_data_entry.findtext('DESCRIPTION/P', ''),
+        }
+        quality_item = quality_item_data_entry.getparent().getparent()
+        study_rob['rob_id'] = quality_item.attrib['ID']
+        study_rob['rob_name'] = quality_item.findtext('NAME')
+        study_rob['rob_description'] = quality_item.findtext('DESCRIPTION/P', '')
+        for group in quality_item.iter('QUALITY_ITEM_DATA_ENTRY_GROUP'):
+            group_id = group.attrib.get('ID')
+            if group_id == study_rob['group_id']:
+                study_rob['group_name'] = group.findtext('NAME')
+        study_robs.append(study_rob)
+
+    # Get references
+
+    included_studies = tree.find('//INCLUDED_STUDIES')
+    for study in included_studies.iter('STUDY'):
+        study_info = {
+            'id': uuid.uuid4().hex,
```

Comment:
Prefer using `uuid.uuid1().hex` for ID fields, as they end up being better for indexing. If you're curious on why, check https://github.com/opentrials/opentrials/issues/337.

I haven't checked in other parts of the code, but what guarantees that if you run this collector multiple times, it won't keep creating new rows in the database?